### PR TITLE
name in parameter mappings of lookup functions isn't necessary

### PR DESCRIPTION
**Pull request by fniederl**

### CODE REVIEW

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 1]
```diff
@@ -411,11 +405,24 @@ private void createLookupTable(final Element maps, final Component transformatio
...
+	private void createParameters(final Component component, final Element componentElement) {
 
 		// TODO: parse parameter values that can be simple string values, JSON objects or JSON arrays (?)
 		// => for now we expect only simple string values
 
+		final String funtionName = component.getFunction().getName();
+
+		if (LOOKUP_FUNCTIONS.contains(funtionName)) {
+
+			final String lookupNameAttr = funtionName.equals(METAMORPH_FUNCTION_LOOKUP) ? "in" : "map";
```
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "in" and "map" could also be a constant here, or? (i.e. private static final ...)

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 2]
```diff
@@ -344,66 +348,56 @@ private void createLookupTable(final Element maps, final Component transformatio
...
+					map.setAttribute("name", component.getName());
 					maps.appendChild(map);
 
 					for (final Entry<String, String> parameterMapping : componentParameterMapping.entrySet()) {
 
-						if (parameterMapping.getKey() != null && parameterMapping.getValue() != null) {
-
-							switch (parameterMapping.getKey()) {
-
-								case "in":
```
Comment (zazi) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "in" and "map" could also be a constant here, or? (i.e. private static final ...)

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 3]
```diff
@@ -344,66 +348,56 @@ private void createLookupTable(final Element maps, final Component transformatio
...
-								case "map":
-
-									map.setAttribute("name", parameterMapping.getValue());
```
Comment (zazi) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "name" could also be a constant here, or? (i.e. private static final ...)

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 4]
```diff
@@ -344,66 +348,56 @@ private void createLookupTable(final Element maps, final Component transformatio
 				if (LOOKUP_FUNCTIONS.contains(component.getFunction().getName()) && componentParameterMapping != null) {
 
 					final Element map = doc.createElement(METAMORPH_ELEMENT_SINGLE_MAP);
+					map.setAttribute("name", component.getName());
```
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "name" could also be a constant here, or? (i.e. private static final ...)

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 5]
```diff
@@ -344,66 +348,56 @@ private void createLookupTable(final Element maps, final Component transformatio
...
-
-												final Element lookup = doc.createElement(METAMORPH_ELEMENT_MAP_ENTRY);
-												lookup.setAttribute("name", lookupEntry);
-												map.appendChild(lookup);
-											}
-										} catch (final IOException e) {
-
-											MorphScriptBuilder.LOG.debug("lookup map as JSON string in parameter mappings could not convert to a list" + e);
+											final Element lookup = doc.createElement(METAMORPH_ELEMENT_MAP_ENTRY);
+											lookup.setAttribute("name", lookupEntry);
```
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "name" could also be a constant here, or? (i.e. private static final ...)

File: converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java [hunk 6]
```diff
@@ -344,66 +348,56 @@ private void createLookupTable(final Element maps, final Component transformatio
...
 
-											for (final Entry<String, String> lookupEntry : lookupEntrys.entrySet()) {
+										for (final Entry<String, String> lookupEntry : lookupEntrys.entrySet()) {
 
-												final Element lookup = doc.createElement(METAMORPH_ELEMENT_MAP_ENTRY);
-												lookup.setAttribute("name", lookupEntry.getKey());
-												lookup.setAttribute("value", lookupEntry.getValue());
-												map.appendChild(lookup);
+											final Element lookup = doc.createElement(METAMORPH_ELEMENT_MAP_ENTRY);
+											lookup.setAttribute("name", lookupEntry.getKey());
```
Comment (boferri) [inline converter/src/main/java/org/dswarm/converter/morph/MorphScriptBuilder.java]: "name" and "value" could also be a constant here, or? (i.e. private static final ...)

**Comment by boferri** [comment]
thx

**Pull request merged**